src/components/map/MapService.js

```diff
@@ -752,7 +752,8 @@ goog.require('ga_urlutils_service');
               }
 
               if (options.zoomToExtent) {
-                olMap.getView().fit(olLayer.getExtent(), olMap.getSize());
+                var extent = olLayer.getSource().getExtent();
```

Comment:
The source can be an `ol.source.ImageVector` which has not a `getExtent` function.

What's wrong with `layer.getExtent()`?